foxy/distribution.yaml

```diff
@@ -1646,7 +1646,7 @@ repositories:
       tags:
         release: release/foxy/{package}/{version}
       url: https://github.com/swri-robotics-gbp/gps_umd-release.git
-      version: 1.0.5-2
+      version: 1.0.6-1
```

Comment:
Can you also update the `version` for the doc and source entries to point to the new `foxy-devel` branch?